Products/Zing/tests/test_datamaps.py

```diff
@@ -47,6 +47,7 @@ def setUp(t):
             isLockedFromUpdates=Mock(return_value=False),
             isLockedFromDeletion=Mock(return_value=False),
         )
+        t.target.device.return_value = t.target
```

Comment:
We need test cases for datamaps that are against components directly, not just for datamaps against devices.  We also need test cases for datamaps against components that are children of components.  We aren't testing much of the functionality here.